MainWindow.cpp

```diff
@@ -29,7 +29,7 @@ void MainWindow::Draw(const Param& param) {
 	/**
 	 * Reposition Window
 	 */
-	ImGuiEx::WindowReposition(param.position, param.cornerVector, param.cornerPosition, param.fromWindowID,
+	ImGuiEx::WindowReposition(nullptr, param.position, param.cornerVector, param.cornerPosition, param.fromWindowID,
```

Comment:
I am not sure, but didn't you wrote somewhere, that the repositioning should be done at the start of the window (next to the `Begin` call)?

Reply:
The way I found to make it work completely without lag is to do it with a hook on ImGuiContextHookType_NewFramePost. Not sure if putting WindowReposition at the start or at the end of the window creation itself makes any difference, I would guess it doesn't

Reply:
Aah, i see, do you already have a working implementation for that?

Reply:
I do yeah, will push to healing stats once this is merged